app/src/main/java/com/b_lam/resplash/util/NotificationManager.kt

```diff
@@ -126,6 +127,7 @@ class NotificationManager(private val context: Context) {
                 GlideApp.with(context).clear(futureTarget)
             }
         }
+        builder.setOngoing(persist == true)
```

Comment:
I would also consider clearing the notification when auto wallpaper gets disabled by the user since they won't be able to do it manually. This could probably be added to the `scheduleAutoWallpaperJob` function in `AutoWallpaperWorker`.